### PR TITLE
Translate release note of 2.0.0-p643 and 2.2.1 (zh_cn)

### PR DESCRIPTION
**Pull request by cassiuschen**
Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia

Please review this pull request. :bowtie:

**Comment by cassiuschen** [comment]
My fault... I forgot to translate the title at first. Now fixed.

File: zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+这个版本的发布包含了许多错误修正。
+请见 [tickets](https://bugs.ruby-lang.org/projects/ruby-200/issues?set_filter=1&amp;status_id=5)
+和 [ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_0_0_643/ChangeLog)
+以了解更多信息.
+
+## 下载
+
+* [http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.bz2](http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.bz2)
+
+      大小:   10772113 字节
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: No need to translate this, thank you!

File: zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      SHA1:   d1d7d324a1f2530d67d54464fe09646583e4dda1
+      SHA256: 1f626f20647693a215a8db3ea0d6ab5ab9cee7c1945cc441b9f8f7b9612b91a0
+      SHA512: 453117152e6facdcd5bedaa9c3b1e349382bc5bc1dd3d650ec58b398cb9d2519a2822d05da10bcc5dbbb4f513fc5fef310caa3529d176fa2d453befb28e4d83a
+
+* [http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.gz](http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.gz)
+
+      大小:   13625546 字节
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: ditto

File: zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      SHA1:   544840583939175886a0885bce1cf07f0b9550b7
+      SHA256: 4bd267a4187e4bc25c1db08f9f9bdc0ce595a705569cac460d98c4f5b02e614e
+      SHA512: cfb88bb4d312861c7c5305593b251648df336f3bf4a4e3f2acc3d66c1a93dc989cf5b60ce9158418ef3fbe4b2e41e7bc86e08942a6624441cfe1297325166b32
+
+* [http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.xz](http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.xz)
+
+      大小:   8318532 字节
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: ditto

File: zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      SHA1:   11568586a6361200efbff33892aaee345625f1f0
+      SHA256: 73f6d939beda8865e12069689ddabd2658b3f637a9adebeee5e374388715c432
+      SHA512: edcff2154eec9c8a84d4c5eac8ffb370e273a5e949923009756cbc069a7ef52de5c91981bd726ae5043bc2784d8ff5080444bc29d0693abc08ff66a8783a7cbc
+
+* [http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.zip](http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.zip)
+
+      大小:   15136932 字节
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: ditto

**Comment by JuanitoFatas** [comment]
@cassiuschen No need to translate the size of each types of download, please change back, thank you!

File: zh_cn/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+author: "hsbt"
+translator: "cassiuschen"
+date: 2015-03-03 03:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布 Ruby 2.2.1 正式发布了！
+该版本是 Ruby 2.2 系列的第一个补丁版本。
+
+这个版本发布包含了对 ffi 编译失败与Symbol GC中内存泄漏问题的修复（详见 [Bug #10686](https://bugs.ruby-lang.org/issues/10686)）。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2015-03-03-ruby-2-2-1-released.md]: **与 Symbol GC 中**

**Comment by cassiuschen** [comment]
Done! @hlcfan @JuanitoFatas

File: zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+我们很高兴地宣布 Ruby 2.0.0-p643 正式发布。
+
+这是 Ruby 2.0.0 的最后一次版本发布。
+Ruby 2.0.0 将进入安全维护阶段。除非出现重大功能性缺失或发现安全问题，将不会再发布后续版本。安全性维护阶段计划持续一年，并计划与2016年2月24日结束。我们建议您准备迁移到新版本（如 2.1 或 2.2）。
+
+
+这个版本的发布包含了许多错误修正。
+请见 [tickets](https://bugs.ruby-lang.org/projects/ruby-200/issues?set_filter=1&amp;status_id=5)
+和 [ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_0_0_643/ChangeLog)
+以了解更多信息.
```
Comment (hlcfan) [inline zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: Period.

File: zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+## 下载
+
+* [http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.bz2](http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p643.tar.bz2)
+
+      SIZE:   10772113 byte
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: It was `bytes`.

File: zh_cn/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+layout: news_post
+title: "Ruby 2.2.1 发布"
+author: "hsbt"
+translator: "cassiuschen"
+date: 2015-03-03 03:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布 Ruby 2.2.1 正式发布了！
+该版本是 Ruby 2.2 系列的第一个补丁版本。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-03-03-ruby-2-2-1-released.md]: This TENNY is important. Read more about it [here](https://www.ruby-lang.org/en/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/). :wink:

**Comment by JuanitoFatas** [comment]
I fixed the period and some minor things, merged manually as f52d647. Thanks for your contribution. Thanks @hlcfan for review! :+1:

File: zh_cn/news/_posts/2015-03-03-ruby-2-2-1-released.md (hunk 8 above)
Comment (cassiuschen) [inline zh_cn/news/_posts/2015-03-03-ruby-2-2-1-released.md]: Oh well thx very much !

Cassius Chen,
2015

**Pull request closed without merging**